packages/office-ui-fabric-react/src/components/Nav/Nav.base.tsx

```diff
@@ -119,15 +118,8 @@ export class NavBase extends React.Component<INavProps, INavState> implements IN
         target={link.target}
         rel={rel}
         disabled={link.disabled}
-        aria-label={
-          link.ariaLabel && selectedStateAriaLabel
-            ? `${link.ariaLabel} ${selectedStateAriaLabel}`
-            : selectedStateAriaLabel
-            ? selectedStateAriaLabel
-            : link.ariaLabel
-            ? link.ariaLabel
-            : undefined
-        }
+        aria-current={isSelectedLink ? 'page' : undefined}
+        aria-label={link.ariaLabel ? link.ariaLabel : undefined}
```

Comment:
`selectedStateAriaLabel` is no longer respected here. This is documented in the `INavLink.types.ts` file as a property but it no longer is applied.

I think that probably makes sense; judging from this crazy logic, it doesn't key off any state and seems to serve no purpose. So a deprecation probably makes sense.

Please modify the comment of the property and replace the description with something like:

```ts
@deprecated Use `ariaLabel` instead.
```

Reply:
reviewed, looks good to me.

Reply:
just wonder if  'page' is enum, does it mean it will work for all languages?

Reply:
@dzearing not allowing authors to create a separate screen-reader-only label was purposeful, since I would consider it an easy-to-misunderstand antipattern. On the other hand, I don't know what the original motivation was; if there's a reasonable need, perhaps hint text/`aria-describedby` could be an alternate place that `selectedStateAriaLabel` could go?

@inateeg it does work for all languages, the localization is handled at the accessibility API and screen reader level. Thanks for checking that :)